src/corelibs/U2Algorithm/src/misc/FindAlgorithm.cpp

```diff
@@ -815,6 +820,10 @@ void FindAlgorithm::find(
     }
 
     try {
+        if (!DynTable::isAcceptableMatrixDimensions(width, height)) {
+            coreLog.error(QObject::tr("The search pattern is too long."));
```

Comment:
Same here

Reply:
'tr': is not a member of 'U2::FindAlgorithm'